apps/server/src/modules/board/controller/mapper/content-element-response.builder.ts

```diff
@@ -0,0 +1,25 @@
+import { NotImplementedException } from '@nestjs/common';
+import { AnyBoardDo } from '@shared/domain';
+import { AnyContentElementResponse } from '../dto';
+import { BaseResponseMapper } from './base-mapper.interface';
+import { FileElementResponseMapper } from './file-element-response.mapper';
+import { TextElementResponseMapper } from './text-element-response.mapper';
+
+export class ContentElementResponseBuilder {
+	private static mappers: BaseResponseMapper[] = [
+		TextElementResponseMapper.getInstance(),
+		FileElementResponseMapper.getInstance(),
+	];
+
+	static mapToResponse(element: AnyBoardDo): AnyContentElementResponse {
+		const mapper = this.mappers.find((mapper1) => mapper1.canMap(element));
```

Comment:
Why not `const elementMapper = this.mappers.find((mapper) => mapper.canMap(element));`?